File: Python/2577.py
list_num = [0,0,0,0,0,0,0,0,0,0]

num1 = 0
num2 = 0
num3 = 0

num1 = int(input())
num2 = int(input())
num3 = int(input())

result = num1 * num2 * num3
result = str(result)

for i in result:
    list_num[int(i)] = list_num[int(i)] + 1

for i in list_num:
    print(i)

File: C++/2577.c
#include <stdio.h>
#define SIZE 10

int main(void) {
	int num1, num2, num3;
	int arr[SIZE] = { 0 };
	scanf("%d", &num1);
	scanf("%d", &num2);
	scanf("%d", &num3);


	int sum = num1 * num2 * num3;

	while (sum) {
		int buff = sum % 10;
		arr[buff]++;
		sum = sum / 10;
	}

	for (int i = 0; i < SIZE; i++) {
		printf("%d\n", arr[i]);
	}
	return 0;
}